### DISCUSSION TITLE
Patient

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#5] [AU Core R1 Resource Priority List - CLOSED]
Upvote for inclusion of an AU Core Resource Profile of [Patient](http://hl7.org/fhir/R4/patient.html) Resource in AU Core Release 1.

Comment if you think we need a different profile to AU Core Patient, or if there is a need for an additional Patient profile in AU Core.

AU Core Patient profile would set minimum expectations for a Patient resource to record, search, and fetch basic demographics and other administrative information about an individual patient. 

AU Core Patient should be based on the [AU Base Patient](http://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-patient.html) profile and identify the additional mandatory core elements, extensions, vocabularies and value sets that SHALL be present in the Patient when conforming to this profile. It provides the floor for standards development for specific uses cases in an Australian context. .

**Comment by reubendaniels** [comment]
I would like to see the following FHIR extensions from the HL7 International _Cross Paradigm Implementation Guide: Gender Harmony - Sex and Gender Representation, Edition 1_ be considered for inclusion as optional data elements in the AU Core Patient profile with localised/AU bindings where possible/appropriate:

— individual-genderIdentity
— patient-sexParameterForClinicalUse
— individual-pronouns
— individual-recordedSexOrGender

**Comment by heathfrankel** [comment]
I would like to see Patient.active to allow false to enable notifications of Patient records that have been deactivated or merged.

**Reply by nicko-123** [reply]
I think patient.active would be derived from the data being transmitted. 

For example, for general practice data, the RACGP definition is used: https://www.racgp.org.au/running-a-practice/practice-standards/standards-5th-edition/standards-for-general-practices-5th-ed/glossary as per definition "A patient who has attended the practice/service three or more times in the past two years".

**Reply by rachelhayhurst** [reply]
Active patient is the subject of much debate - its definition is fraught without sufficient agreed criteria to define what is an encounter - it is calculated differently in almost every scenario using GP data and often not defined by the clinical information systems themselves. An administrative date stamp of a data value is often used which may not equate to an encounter - this means some patients are RACGP active without having been in the practice in the las two years (eg the '3 times' date stamps may come from recalls and reminders). Occasion of service should be defined: sometimes 3 attendances are used regardless of the attendance date (so you can be RACGP active with three encounters on one day, eg, one for the GP consult, one for wound care and one for a vaccination) and in others its unique visits ie across three separate dates. There are also challenges and differences defining a clinical encounter and whether any admin (non client contact) should be counted as an encounter. Plenty more examples of the complexity including encounters recorded in the clinical system by visiting specialist staff.
